tensorflow/contrib/lite/java/AndroidManifest.xml

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="org.tensorflow.lite">
-    <application>
-    </application>
-</manifest>
+    package="org.tensorflow.lite">
+
+    <uses-sdk
+        android:minSdkVersion="4"
+        android:targetSdkVersion="28" />
```

Comment:
Can you change this to targetSdkVersion=19? Specifying 28 can have some undesirable side-effects for certain clients who don't actually want to target such a recent API.

Alternatively, I'm happy to make this change directly in the branch.

Reply:
Actually, why not just removing the targetSdkVersion? Then it would default to the minSdkVersion.